clusterloader2/testing/load/daemonset.yaml

```diff
@@ -1,4 +1,5 @@
 {{$Image := DefaultParam .Image "k8s.gcr.io/pause:3.1"}}
+{{$AutomountServiceAccountToken := DefaultParam .AutomountServiceAccountToken true}}
```

Comment:
Can we do instead:
```
{{$AutomountServiceAccountToken := DefaultParam .CL2_LOAD_DAEMON_SETS_AUTOMOUNT_SERVICE_ACCOUNT_TOKEN true}}
```

And revert all the changes in config.yaml?

The rule of thumb I'd like to apply here to make the load config easier to read (mostly shorter): use `templateFillMap` if test uses same template with different configs, e.g. `CpuRequest` is a good example in density test. Otherwise, create `CL2_*` env var inside the template to make the config.yaml shorter.

Reply:
+1

I also don't see particular value in allowing to configure mounting SA`s differently for different objects. Rather - we could set some percentage-targets controlled by a single thing (i.e. ~X% of all pods that we create will have default SA mounted).

Reply:
> And revert all the changes in config.yaml?

Sure, make sense. I didn't know that envs are passed to all object templates.

> I also don't see particular value in allowing to configure mounting SA`s differently for different objects.

It seems that DaemonSet should be considered in a different way - we are passing template only for a single namespace, to this percentage will have a huge impact on a performance test.

Also, other Pods are no uniform - for STS we are mounting PVC, for deployment we are mounting configmap/secret. I do not know if this impacts performance of watches.

> Rather - we could set some percentage-targets controlled by a single thing (i.e. ~X% of all pods that we create will have default SA mounted).

Sure, I can change bool flag to % flag, however I still prefer to keep them divided per kind.